integration/kubernetes/runtimeclass_workloads/pod-block-pv.yaml

```diff
@@ -7,7 +7,7 @@ spec:
   runtimeClassName: kata
   containers:
     - name: my-container
-      image: alpine
+      image: quay.io/footloose/ubuntu18.04:latest
```

Comment:
why this image and not the image from docker.io?

Reply:
That's due to me mentioning the size difference is huge. Then @bergwolf go to a 18.04 image we already have.
But, hey, let me change my mind. Let's use the one from docker.io, everywhere.

@bergwolf, sorry for the noise. /o\

Reply:
because this one is already used by other bats, so that we do not have to pull yet another image.

Reply:
@bergwolf, feel free to get back to using the default one from docker.io, my bad.

Later on, in the future, we can try to standardize those. As I said, sorry for the noise.

Reply:
nvm, I've restored the original commit ;)